### PR TITLE
implemented 1283, also the code for permissions in widgets must not c…

### PR DESCRIPTION
**Pull request by boutell**
…rash if there is no manager for a widget

### CODE REVIEW

**Comment by boutell** [comment]
See #1283

File: lib/modules/apostrophe-express/index.js [hunk 1]
```diff
@@ -353,8 +353,12 @@ module.exports = {
       // Allow the options for bodyParser to be customized
       // in app.js
       var bodyParserOptions = _.extend({
-        json: { },
-        urlencoded: {extended: true}
+        json: {
```
Comment (austinstarin) [inline lib/modules/apostrophe-express/index.js]: Is this related?

Comment (boutell) [reply lib/modules/apostrophe-express/index.js]: Indeed, this is the piece that *is* related (to 1283, the issue in question). Plus per the title of the PR there's one other minor fix in the other spot.
